multinet/types.py

```diff
@@ -1,10 +1,25 @@
 """Custom types for Multinet codebase."""
-from typing import Dict, Set
+from typing import Dict, Set, List
 from typing_extensions import Literal, TypedDict
 
 EdgeDirection = Literal["all", "incoming", "outgoing"]
 TableType = Literal["all", "node", "edge"]
 
+WorkspacePermissions = TypedDict(
+    "WorkspacePermissions",
+    {
+        "owner": str,
```

Comment:
To clarify, this maps to the `sub` field of a user? If so, it might be good to add a comment here to explain that.

Reply:
See ca96a922d26c192cce8fbcada77024cca62196e6.